app/src/main/java/com/example/mona/facebookoffline/ActivityRequestCode.java

```diff
@@ -0,0 +1,9 @@
+package com.example.mona.facebookoffline;
+
+/**
+ * Created by mona on 5/4/16.
+ */
+public class ActivityRequestCode {
+
+    public static final int SELECT_PHOTOS = 100;
```

Comment:
I like having a separate class here so that it's easy to verify that an activity request code isn't duplicated. In the future when we maybe write unit tests, we could easily write a small test to verify this. That said, I don't have a strong opinion for right now